libs/components/text-editor/src/lib/modules/text-editor/text-editor.component.spec.ts

```diff
@@ -1333,6 +1334,10 @@ describe('Text editor', () => {
     }));
 
     it('should set the style of the iframe body to the provided style state', fakeAsync(() => {
+      // We normally load the iframe in the `beforeEach`. However, we need to reset things here so that the initial styles can be applied first.
```

Comment:
This is where the `setupTest` function would be useful (future consideration). Thanks for the comment.